cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/FileType.java

```diff
@@ -21,17 +21,115 @@
  * Enum which holds the file type: service or main.
  */
 public enum FileType {
-    SERVICE("hello_service.bal"),
-    MAIN("main.bal");
+    SERVICE("hello_service.bal", "// A system package containing protocol access constructs\n" +
```

Comment:
Can we move this content to a string variable and refer it here. In that case, its more readable.

Reply:
Fixed with 0415ac7